renderer/pages/home.tsx

```diff
@@ -1,12 +1,14 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Head from 'next/head';
 import Link from 'next/link';
-import { Button, Heading } from 'rebass';
+import { Button, Heading, Text } from 'rebass';
+import SelectFolder from '../components/select-folder';
 
 import useIpcService from '../hooks/useIpcService';
 
 export default function Home() {
     const ipcService = useIpcService();
+    const [folder, setFolder] = useState();
 
     const analyze = async () => {
         if (!ipcService) return;
```

Comment:
@julesyoungberg Originally put the folder in here on your updated branch but kept it out for now.

https://github.com/julesyoungberg/soundboy/pull/4/files#diff-3b06ade841921657f86f369165e5fea9R16

Reply:
sounds, good, avoids conflict resolution this way